### PR TITLE
Polkadart amend

### PR DESCRIPTION
**Pull request by leonardocustodio**
Lohann has done some amazing work into this lib but unfortunately he has not been able to dedicate the necessary time to continue with the works. 

As discussed in the Grant PR proposal https://github.com/w3f/Grants-Program/pull/1053#issuecomment-1720243331 to continue the development and finish this grant we opted to change the project owner as well as the milestone values and objectives.

We acknowledge we underestimated the effort required to finish the last two milestone. There were 5 engineers working on this but at this stage things got too complex and we remained with only 2 engineers with enough knowledge to finish the work.

Polkadart has been an amazing project and we have already have people using it in production environment. As Flutter together with React Native is one of the most popular mobile frameworks we are very sure that this project is a great contribution to our ecosystem allowing people to develop apps that support substrate based chains in any language they want.

### CODE REVIEW

**Comment by leonardocustodio** [comment]
I have read and hereby sign the Contributor License Agreement.

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks for the PR. Could you also update the total costs?

**Comment by semuelle** [comment]
For reference, Lohann has given his approval [here](https://github.com/w3f/Grants-Program/pull/1053#issuecomment-1732368046).

File: applications/Polkadart.md [hunk 1]
```diff
@@ -182,23 +161,54 @@ Described above.
...
+| 0b.  | Documentation    | We will provide markdown documentation of how to install and use the API. |
+| 0c.  | Testing Guide    | We will provide sample scripts on how to use it. |
+| 0d.  | Docker           | We will provide a Dockerfile(s) that can be used to test the PoC. |
+| 0e.  | Article/Workshop | We will publish an example flutter project using the library to explain what was done and how to use it. |
+| 3a.  | Substrate Uri    | Implement Substrate's standard derivation format that applies to seeds and mnemonics. |
+| 3b.  | Encrypt/Decrypt  | Use Substrate's Bip39 password to encrypt/decrypt seeds and mnemonics. |
+| 3c.  | Ed25519          | Support Ed25519 signatures and key derivation |
+| 3e.  | Sign & Verify    | Sign and verify messages using ed25519 keypairs |
+| 3f.  | Substrate Signed Extensions | Support substrate's signed extensions |
+| 3g.  | Statemint Signed Extensions | Support statemint's signed extensions |
```
Comment (semuelle) [inline applications/Polkadart.md:182]:
```suggestion
| 3g.  | AssetHub Signed Extensions | Support AssetHub's signed extensions |
```

**Review by semuelle — CHANGES_REQUESTED** [review]
Thanks for the amendment, @leonardocustodio. I'm generally happy to approve it. Could you clarify the timeline, though? You are, if I'm reading this right, planning to submit M3 in six months and requesting 40k for 2 full-time developers for the next ten months?

**Comment by leonardocustodio** [comment]
Hello @semuelle,

Sure, milestone 3 is already halfway through I've increased it to 6 months to consider the time that was already spent and to have the project proposal updated with the whole time spent and the whole investment made. The delivery of milestone 3 should happen in 2 to 3 months. So I would say we "only" have about 6 months ahead to finish the whole project.

**Review by semuelle — APPROVED** [review]
Thanks. I think the added scope seems quite useful, so happy to approve.

**Review by keeganquigley — APPROVED** [review]
Thanks @leonardocustodio we appreciate it, LGTM.

**Review by Noc2 — APPROVED** [review]

**Review by takahser — APPROVED** [review]

**Review by dsm-w3f — APPROVED** [review]

**Pull request merged**